### ISSUE TITLE
Text fails to meet minimum requirement contrast ratio (4.5:1) with background for regular text.  Impact on Users: This will impact low vision users. People with low vision often have difficulty reading text that does not contrast with its background. This can be exacerbated if the person has a color vision deficiency that lowers the contrast even further.  This problem is located at Settings window on the left side list elements. The list of elements has a contrast ratio 3.2:1 that is less than 4.5:1 for the minimum ratio requirement. And in the “Dark” settings of Theme, the contrast ratio is just below 4.5:1.

### ISSUE THREAD

**Issue by olgatolk71** [dequelabs/axe-core#4573]
### Product

axe-core

### Latest Version

- [X] I have tested the issue with the latest version of the product

#### Expectation
Describe what you expected the product to do.

#### Actual
Describe what the product actually does.

#### How to Reproduce
Provide a code sample or link to a webpage that reproduces the issue. Without this, your issue may be closed without investigation.

#### Additional context
Any thing else we should know about the issue?

**Comment by straker** [comment]
Thanks for the issue. Could you provide more information? Specifically could you fill out each section of the issue description, otherwise we'll have to close the issue.
